gizmos/tree.py

```diff
@@ -30,17 +30,29 @@
 
 
 def main():
```

Comment:
Hi, I noticed this just now. Small comment regarding what I assume are some formatting changes due to `black`. Generally I've been keeping the default options, but one exception is line length. I've been using `black` with a modified line length of 100 chars instead of the default 88 (see https://github.com/psf/black). I find that 100 characters makes the code easier to read.

Reply:
Yes, 100 characters is our policy, recorded here: https://github.com/knocean/practises/tree/master/python#style but maybe not completely consistent yet.